packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1478,9 +1480,9 @@ curl -v -X GET \
 
 #### List Users with search
 
-Searches for users based on the properties specified in the search parameter
+Searches for users based on the properties specified in the search parameter. To ensure optimal performance, we recommend using the search parameter over filter.
 
-> **Note:** Listing users with search should not be used as a part of any critical flows, such as authentication, to prevent potential data loss. Search results may not reflect the latest information, as this endpoint uses a search index which may not be up-to-date with recent updates to the object.
+> **Note:** Results from the Search API are computed from asynchronously indexed and eventually consistent data. The indexing delay is typically less than one second.
```

Comment:
Results from the Search API are eventually consistent and typically converge in less than one second.

Reply:
The results are based on eventually consistent data.  The results are themselves not eventually consistent.